src/test/java/org/javarosa/core/model/utils/test/QuestionPreloaderTest.java

```diff
@@ -0,0 +1,41 @@
+package org.javarosa.core.model.utils.test;
+
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.javarosa.core.util.BindBuilderXFormsElement.bind;
+import static org.javarosa.core.util.XFormsElement.body;
+import static org.javarosa.core.util.XFormsElement.head;
+import static org.javarosa.core.util.XFormsElement.html;
+import static org.javarosa.core.util.XFormsElement.input;
+import static org.javarosa.core.util.XFormsElement.mainInstance;
+import static org.javarosa.core.util.XFormsElement.model;
+import static org.javarosa.core.util.XFormsElement.t;
+import static org.javarosa.core.util.XFormsElement.title;
+
+import java.io.IOException;
+import org.javarosa.core.test.Scenario;
+import org.javarosa.xform.parse.XFormParser;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class QuestionPreloaderTest {
+    @Ignore("Not supported")
```

Comment:
I think I'd generally prefer a test that documents the current behaviour (with a comment to illustrate that it's maybe not ideal) instead of an ignored one that documents a possible future (or here maybe "expected" behaviour). This test is great as a piece of documentation, but it won't give us any warning/indication if there's a change in behaviour because of the `@Ignore`.